src/LiveComponent/doc/index.rst

```diff
@@ -352,6 +352,12 @@ clicked). To do that, use ``norender`` modifier:
 
     <input data-model="norender|max">
 
+You might as well deref for all fields ``*`` of the form:
```

Comment:
```suggestion
You can also defer re-rendering all fields for the form using ``*``:
```

Reply:
Agreed, will rework this

Reply:
I've used your sentence, but did not understand `move this section down to the end`
after which word you want it please?